pallets/pips/src/lib.rs

```diff
@@ -837,7 +867,7 @@ decl_module! {
         /// # Errors
         /// * `BadOrigin`: Only the owner of the proposal can amend it.
         /// * `ProposalIsImmutable`: A Proposal is mutable only during its cool off period.
-        #[weight = (750_000_000, DispatchClass::Operational, Pays::Yes)]
+        #[weight = <T as Trait>::WeightInfo::cancel_proposal(1_000)]
```

Comment:
Proposal padding is not very relevant here. Let's just remove this variable and use a constant length in the benchmarks.